test/releasers/fixtures/dotnet/commits.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
Is this test data?

Reply:
Yes, this is what the response from the GitHub V4 commits API looks like. It basically represents a fake git history, actually not dissimilar to the type of testing data you'd use in `synthtool`.